Features/create ubiquity os user manual (#94)
ubiquity/business-development: UbiquityOS-User-Manual/developer_onboarding/getting-started/step-by-step-onboarding/setup-devpool-account.md

# Setup DevPool Account

## Create GitHub Account and Register to DevPool

Visit  [GitHub Official Website](https://github.com/) and click on "**Sign Up**" if you don\`t have account or if you have already have a GitHub account click on "**Log in**" to get stand with GitHub. Complete the registration form to create your account.

* **Verify Your  GitHub Account**

Follow the prompts to verify your account. Secure your account with two-factor authentication.

Then, go to the [devpool directory](https://devpool.directory/) and log in with your GitHub account.

<figure><img src="../../../.gitbook/assets/image (6) (1).png" alt=""><figcaption></figcaption></figure>

Next, GitHub prompt you to register to the [DevPool](https://devpool.directory/).&#x20;

<div align="center"><figure><img src="../../../.gitbook/assets/image (1) (1) (1) (1) (1) (1) (1) (1) (1).png" alt="" width="322"><figcaption></figcaption></figure></div>

Then, fill out the GitHub credentials, such as **username** and **password**.

<figure><img src="../../../.gitbook/assets/image (2) (1) (1) (1) (1) (1).png" alt="" width="375"><figcaption></figcaption></figure>

> Finally, authenticate the Ubiquity to access your personal data for registration.

Reviewer: ```suggestion
Finally, allow our GitHub App to access your personal data for registration.
```
